test/playback.test.js

```diff
@@ -19,7 +19,7 @@ const playFor = function(player, time, cb) {
 
   const checkPlayerTime = function() {
     window.setTimeout(() => {
-      if (player.currentTime() <= targetTime) {
+      if (player.tech_ && player.tech_.el_ && player.currentTime() <= targetTime) {
```

Comment:
This prevents an error from being logged when this functoin runs after the player has disposed on a test timeout.